posts/2020-12-07-the-foundation-conversation.md

```diff
@@ -0,0 +1,67 @@
+
+## How to join the conversation
+
+There are many ways to participate in the Foundation Conversation:
+
+* **[Read the draft FAQ we’ve been working on][FAQ]**. It contains the answers to some of the questions that we have been asked thus far.
+* **Fill out our [survey]**. This survey is designed to help us understand how the Rust community is feeling about the Foundation.
+* **Ask questions during the Community Q&A periods**. We’ve scheduled a number of 3 hour periods during which the foundation-faq-2020 repo will be open for anyone to ask questions. There will be members of the core team around during those periods to answer those questions as best we can.
+* **Watch our Live Broadcasts**. We’ve scheduled live broadcasts this week where members of the core team will be answering and discussing some of the questions that have come up thus far. These will be posted to YouTube later.
+
+Read on for more details.
+
+## The foundation-faq-2020 repository
+
+We have chosen to coordinate the Foundation Conversation using a GitHub repository called [foundation-faq-2020]. This repository contains the [draft FAQ][FAQ] we’ve written so far, along with a series of issues representing the questions that people have. Last week we opened the repository for Rust team members, so you can see that we’ve already had quite a few questions raised (and answered). Once a new issue is opened, someone from the core team will come along and post an answer, and then label the question as “[answered]”.
+
+## Community Q&A sessions
+
+We have scheduled a number of 3 hour periods in which the repository will be open for anyone to open new issues. Outside of these slots, the repository is generally “read only” unless you are a member of a Rust team. We are calling these slots the “Community Q&A” sessions, since it is a time for the broader community to open questions and get answers.
+
+We’ve tried to stagger the times for the “Community Q&A” periods to be accessible from all time zones. During each slot, members of the core team will be standing by to monitor new questions and post answers. In some cases, if the question is complex, we may hold off on answering right away and instead take time to draft the response and post it later.
+
+Here are the times that we’ve scheduled for folks to pose questions.
+
+|                                                 | PST US | EST US | UTC Europe/Africa  | India                   | China            |
+|-------------------------------------------------|--------|--------|--------------------|-------------------------|------------------|
+| Dec 7th ([View in my timezone][dec7-session])   | 3-6pm  | 6-9pm  | 23:00-2:00 (Dec 8) | 1:30am-4:30am (Dec 8)   | 3pm-6am (Dec 8)  |
```

Comment:
I think there was a typo in the Chinese timezone, it should be 3*a*m-6am rather than 3pm-6am, right @ashleygwilliams

Reply:
@Mark-Simulacrum

Reply:
you are correct @RageKnify! i will fix